frontend/dockerfile/docs/reference.md

```diff
@@ -2044,6 +2357,18 @@ ARG TARGETPLATFORM
 RUN echo "I'm building for $TARGETPLATFORM"
 ```
 
+### BuildKit built-in build args
+
+| Arg                                   | Type    | Description                                                              |
+|---------------------------------------|---------|--------------------------------------------------------------------------|
+| `BUILDKIT_CACHE_MOUNT_NS`             | String  | Set optional cache ID namespace.                                         |
```

Comment:
Do we have more details on these that we can link to ? (some example that describes how they're used)

Reply:
Added a simple example for one of them. Let me know if it lgty.